eth/beacon/types/blocks.py

```diff
@@ -111,3 +113,17 @@ def root(self) -> Hash32:
     @property
     def num_attestations(self) -> int:
         return len(self.body.attestations)
+
+    @classmethod
+    def create_block_without_signature_root(
```

Comment:
Whoops, sorry, now I think this function could just return the root as its name says.
```python
    @classmethod
    def get_block_without_signature_root(
            cls,
            block: 'BaseBeaconBlock') -> Hash32:
        return cls(
            slot=block.slot,
            parent_root=block.parent_root,
            state_root=block.state_root,
            randao_reveal=block.randao_reveal,
            candidate_pow_receipt_root=block.candidate_pow_receipt_root,
            signature=EMPTY_SIGNATURE,
            body=block.body,
        ).root
```